### ISSUE TITLE
Failed to build the the docker image

### ISSUE THREAD

**Issue by xuwang** [purpleworks/fleet-ui#12]
Trying to build the docker image by following the README. Somehow the angular/dist folder never gets created/built, and the following step would fail because angular/dist doesn't exit:

ADD angular/dist /root/fleet-ui/public

The build environment is wheeze, BTW,  if that matters. 

Any idea why?

**Comment by DorianGray** [comment]
I had the same issue. You need to do several things that are not in the README. Run grunt build in the angular folder.

**Comment by xuwang** [comment]
@DorianGray you are right. There a lot more steps depending on what platform one uses to build, but we have docker come to rescue!  I built a docker-fleetui-builder docker which is an image contains all necessary tools including docker daemon and use that to build the 'fleet-ui". See README here:

https://github.com/xuwang/docker-fleetui-builder

I have tested it with a fork of purpleworks/fleet-ui.  Should work because the docker image doesn't depend on what OS host you are on.
